### DISCUSSION TITLE
WeMod Issues with some games

### DISCUSSION THREAD

**Discussion by qnlbnsl** [DeckCheatz/wemod-launcher#93] [General]
So as the title says I'm having some issues with a few games here and there where either wemod would never show up (it is running in the background) or it would take over the display (game is in the background)
In both cases one or the other application must be closed before i can see the game or wemod. 

So, if wemod is in the foreground then switching to the game window does not work. I have to close wemod before i can see teh game window

If it is in the background, then i must quit the game before i see wemod. 

Is there anything i can do to debug the issue?

**Comment by marvin1099** [comment]
Wemod will keep running in the backround if you close it,
You can reopen it from the system tray,
if you are on steamos enable devoper mode then you can switch windows using the steam button,
On a normal Linux desktop you can switch windows by using alt+tab, check your ditro info for that if it won't work

**Reply by qnlbnsl** [reply]
This is so true. I can't even think of why I never co spidered the close button in WeMod as minimizing it as I was thinking that the steamOS window (in the game mode) is the process itself. 

I did as you suggested and it worked!! Thanks for the help!

**Comment by marvin1099** [comment]
Good
